packages/core/upload/admin/src/components/AssetCard/AudioAssetCard.js

```diff
@@ -34,6 +34,10 @@ const AudioPreviewWrapper = styled(Box)`
   }
 `;
 
+const StyledCard = styled(Card)`
+  height: 100%;
```

Comment:
<img width="490" alt="image" src="https://user-images.githubusercontent.com/71838159/172387052-f2e8fd11-fa2a-43c1-948f-5910191f0e1e.png">

Card is actually a Box so we should be able to just pass the `height="100%"` props to it!